### PR TITLE
my.nextdns.io fixes

### PR DESCRIPTION
**Pull request by FadeMind**
https://my.nextdns.io/ `<configid>` /settings

### CODE REVIEW

**Comment by FadeMind** [comment]
Don't touch flags in analytics tab

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -12820,6 +12820,7 @@ img[src*="/static/media/sonos"]
 .d-flex > .text-break > img
 .d-flex > .flex-grow-1 > div > img
 img[src*="handshake-logo"]
+.d-flex.align-items-center.flex-wrap a img
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:12823]: What about appending `:not(img[src*="/misc/flags/"] )`?

Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config:12823]: `:not(img[src*="/misc/flags/"]` breaks icons

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:12823]: Which selector is breaking them? Try to fix that selector instead of manually reverting the filter.

Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config:12823]: You mixed things IMHO.
`.d-flex.align-items-center.flex-wrap a img` is intended to invert directly logos in Web3 section. 

CSS rule is for revert to default generic colors flags. 

Your idea breaks all site (added in new line standalone)

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:12823]: > CSS rule is for revert to default generic colors flags.

Is there a filter by the site that is affecting these flags?

Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config:12823]: Mixing with 
```
.d-md-inline.ml-md-3.mt-1.mt-md-0
span.notranslate
```
invert part of text and breaks things visible. My workaround with CSS hack do this properly, but infact it is ugly.

**Review by Gusted — APPROVED** [review]

**Pull request merged**